src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.model.dto.request.UserRequestDto;
+import mate.academy.spring.model.dto.response.UserResponseDto;
+import mate.academy.spring.security.AuthenticationService;
+import mate.academy.spring.service.dto.mapping.impl.response.UserResponseMapper;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AuthenticationController {
+    private final AuthenticationService authenticationService;
+    private final UserResponseMapper userResponseMapper;
```

Comment:
let's use abstraction instead of implementation here
```suggestion
    private final DtoResponseMapper<UserResponseDto, User> userResponseMapper;
```